python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1042,10 +1042,34 @@ def execute_asset_backfill_iteration(
             )
             instance.update_backfill(updated_backfill)
 
+        new_materialized_partitions = (
+            updated_backfill_data.materialized_subset
+            - previous_asset_backfill_data.materialized_subset
+        )
+        new_failed_partitions = (
+            updated_backfill_data.failed_and_downstream_subset
+            - previous_asset_backfill_data.failed_and_downstream_subset
+        )
+        new_requested_partitions = (
+            updated_backfill_data.requested_subset - previous_asset_backfill_data.requested_subset
+        )
         logger.info(
             f"Asset backfill {updated_backfill.backfill_id} completed iteration with status {updated_backfill.status}."
         )
         logger.info(
+            "Backfill iteration summery:\n"
```

Comment:
still trying to come up with headings for these two log messages. I think having a summary of what changed during this tick of the daemon is useful and having an update on the cumulative status of the backfill is also useful, but i'm not sure how to distinguish the two in logs 

Ideas:
For the iteration summary: trying to communicate "this is what changed during this iteration"
- Backfill iteration summary 
- Backfill iteration diff
- Updates from this iteration of the backfill daemon 

For the entire backfill summary: trying to communicate "this is the overall status of the backfill"
- Backfill summary 
- Cumulative backfill status
-